bochs/bx_debug/dbg_main.cc

```diff
@@ -1245,56 +1251,96 @@ void bx_dbg_info_registers_command(unsigned cpu, int which_regs_mask)
 #endif
 #if BX_SUPPORT_X86_64 == 0
     reg = BX_CPU(cpu)->get_reg32(BX_32BIT_REG_EAX);
-    dbg_printf("eax: 0x%08x %d\n", (unsigned) reg, (int) reg);
+    dbg_printf("eax: 0x%08x %d ", (unsigned) reg, (int) reg);
```

Comment:
awful code duplication.
should be array of const char* containing "rax", "rbx", ... , "r15" like this one from disasm.cc:

static const char *general_64bit_regname[17] = {
    "rax", "rcx", "rdx", "rbx", "rsp", "rbp", "rsi", "rdi",
    "r8",  "r9",  "r10", "r11", "r12", "r13", "r14", "r15", "rip"
};

and for walking over all registers